webviews/components/ProjectInfo.svelte

```diff
@@ -1,118 +1,166 @@
 <script>
-  import { gql } from "@apollo/client";
-  import { query } from "svelte-apollo";
-
-  const GET_REPO_PROJECT_INFO = gql`
-    query GetRepoProjectInfo($name: String!, $owner: String!, $number: Int!) {
-      repository(name: $name, owner: $owner) {
-        project(number: $number) {
-          name
-          body
-          columns(first: 100) {
-            nodes {
-              name
-              cards(first: 100) {
-                nodes {
-                  content {
-                    ...fieldsIssue
-                    ...fieldsPR
-                  }
-                  note
-                  isArchived
-                  state
-                }
-              }
-            }
-          }
-        }
-      }
-      rateLimit {
-        limit
-        cost
-        remaining
-        resetAt
-      }
-    }
-
-    fragment fieldsIssue on Issue {
-      title
-    }
-
-    fragment fieldsPR on PullRequest {
-      title
-    }
-  `;
-
-  const GET_ORG_PROJECT_INFO = gql`
-    query GetOrgProjectInfo($login: String!, $number: Int!) {
-      organization(login: $login) {
-        project(number: $number) {
-          name
-          body
-          columns(first: 100) {
-            nodes {
-              name
-              cards(first: 100) {
-                nodes {
-                  content {
-                    ...fieldsIssue
-                    ...fieldsPR
-                  }
-                  note
-                  isArchived
-                  state
-                }
-              }
-            }
-          }
-        }
-      }
-      rateLimit {
-        limit
-        cost
-        remaining
-        resetAt
-      }
-    }
-
-    fragment fieldsIssue on Issue {
-      title
-    }
-
-    fragment fieldsPR on PullRequest {
-      title
-    }
-  `;
+  import { mutation, query } from "svelte-apollo";
+  import * as queries from "./queries.js";
 
   export let type, name, owner, login, number;
 
   const projectInfo =
     type === "repo"
-      ? query(GET_REPO_PROJECT_INFO, {
+      ? query(queries.GET_REPO_PROJECT_INFO, {
           variables: {
             name: name,
             owner: owner,
             number: parseInt(number),
           },
-          pollInterval: 1800,
+          pollInterval: 100,
```

Comment:
I think it's best to keep this a bit higher; GitHub has a rate limit, so we can't fetch data too often

Reply:
I was _just_ reading some blog that GitHub will ban you if you keep calling their APIs too often.
So +1, I'll edit it back :p

Reply:
It looks blazingly fast when we poll after every 100ms though